scan-camera/src/main/java/com/getbouncer/scan/camera/camera1/Camera1Adapter.kt

```diff
@@ -109,18 +109,22 @@ class Camera1Adapter(
         }
     }
 
-    @ExperimentalCoroutinesApi
```

Comment:
nothing here was experimental, this is left-over from an older commit, so remove it.